src/app/containers/Text/index.jsx

```diff
@@ -1,9 +1,10 @@
 import React from 'react';
 import paragraph from '../Paragraph';
+import unorderedList from '../BulletedList';
 import Blocks from '../Blocks';
 import { textModelPropTypes } from '#models/propTypes/text';
 
-const componentsToRender = { paragraph };
+const componentsToRender = { paragraph, unorderedList };
```

Comment:
Does this affect articles?  We only want to render these bulleted lists in CPS pages

Reply:
Moved to a new component